CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Is CMake 3.7 really required? I am not sure if we want to raise the minimal required CMake version.

Reply:
not really

Reply:
sorry for the timing i still have some changes in my mac will push that asap